packages/badge/stories/badge.stories.tsx

```diff
@@ -36,47 +36,31 @@ const shapes = {
 
 export default {
   title: "Graphic Elements/Badge",
-  component: Badge.type,
-  subcomponents: { BadgeButton, ColorCodedBadge },
+  component: Badge,
+  subcomponents: { BadgeButton },
   decorators: [
-    withKnobs,
     Story => (
       <div style={{ margin: "0 3em" }}>
         <Story />
       </div>
     )
-  ]
-};
-
-export const Default = args => <Badge {...args}>Default</Badge>;
-
-export const Primary = () => <Badge appearance="primary">Primary</Badge>;
-
-export const Success = () => <Badge appearance="success">Success</Badge>;
-
-export const Danger = () => <Badge appearance="danger">Danger</Badge>;
-
-export const Warning = () => <Badge appearance="warning">Warning</Badge>;
-
-export const Outline = () => <Badge appearance="outline">Outline</Badge>;
-
-export const DefaultBadgeButton = () => (
-  <BadgeButton onClick={action("badge button")}>Badge Button</BadgeButton>
-);
+  ],
+  argTypes: {
+    appearance: {
+      options: [
+        "default",
+        "primary",
+        "success",
+        "warning",
+        "danger",
+        "outline"
+      ],
+      control: { type: "select" }
+    }
+  }
+} as Meta;
 
-export const DefaultColorCodedBadge = () => (
-  <ColorCodedBadge color={select("Color", colors, blue)}>
-    Color Coded Badge
-  </ColorCodedBadge>
-);
+const Template: Story = args => <Badge {...args}>Badge</Badge>;
 
-export const IconColorCodedBadge = () => (
-  <>
-    <ColorCodedBadge
-      color={select("Color", colors, blue)}
-      iconShape={select("Shape", shapes, SystemIcons.CircleCheck)}
-    >
-      Color Coded Badge with Icon
-    </ColorCodedBadge>
-  </>
-);
+export const Default = Template.bind({});
```

Comment:
nice consolidation of stories here 👍🏻

Reply:
I am noticing that this doesn't have a display name on the code page. This may be expected but thought I'd point it out:

<img width="1084" alt="Screen Shot 2021-12-07 at 11 14 49 AM" src="https://user-images.githubusercontent.com/815936/145075619-253e1d73-334a-407b-ae46-484cd8d13d43.png">

Reply:
Thank you for finding this 🕵️‍♂️  I found a fix. This is an issue storybook and the docs addon has with the name/displayName of components.